doc-content/enterprise-only/ai/ai-con.adoc

```diff
@@ -0,0 +1,55 @@
+
+COMMENT: Jess, graphic from slide one, red building blocks (just the blocks, not the heading)
+
+While science fiction is filled with what is referred to as artificial general intelligence (AGI), machines that perform better than people and cannot be distinguished from them and learn and evolve without human intervention or control, AGI is decades away. Meanwhile, we have pragmatic AI which is a lot less frightening and much more useful to us today. Pragmatic AI is a collection of AI technologies that combined provide solutions to problems such as predicting customer behavior, providing automated customer service, and helping customers make purchasing decisions.
+
+Leading industry analysts report that previously organizations have struggled with AI technologies because they invested in the potential of AI rather than the reality of what AI can deliver today. AI projects were not productive and as a result investment in AI slowed and budgets for AI projects were reduced. This disillusionment with AI is often referred to as an AI winter. AI has experienced several cycles of AI winters followed by AI springs and we are now decidedly in an AI spring. Organizations are seeing the practical reality of what AI can deliver. Being pragmatic means being practical and realistic.  A pragmatic approach to AI considers AI technologies that are available today, combines them where useful, and adds human intervention when needed to create solutions to real-world problems.
+
+.Pragmatic AI solution example
+////
+Note: Keeping for now.
+A major transportation company has a fleet of trains and a network of train tracks. The company uses machine learning to identify geographic clusters where trains reported some type of problem. If a train reports a problem when it passes a specific position on the track it is recorded as data and if enough problems are recorded in the same location, you can identify a cluster. But that information is not complete enough to be useful so you can add digital decision to filter the data. For example, you can say that a failure must be a specific type of failure and must be reported by at least three trains. If that condition it met, you can use maths optimization to schedule a repair crew to investigate the track. Or if only one train has the same type of problem more than once at the location, you can use maths optimization to schedule the train for repair.
+////
+
+One application of pragmatic AI is in customer support. A customer files a support ticket that reports a problem, for example a login error. A machine learning algorithm is applied to the ticket to match the ticket content with existing solutions, based on keywords or natural language processing (NLP). The keywords might appear in many solutions, some relevant and some not as relevant. You can use digital decisioning to determine which solutions to present to the customer.
```

Comment:
this new example however is missing "Math Optimization" part...

Reply:
Good point, I think we can add a similar sentence to the example

If none of the answers proposed by the algorithm is good enough to be proposed (i.e. low confidence score or multiple answers with high confidence), the outcome of the digital decisioning could be to involve support team. At this point mathematical optimization selects the best assignee for the ticket considering employee rostering constraints

Wdyt?